### PR TITLE
Invoice creation

### PR DESCRIPTION
**Pull request by chimedpalden**

### CODE REVIEW

File: spec/harvesting/harvest_data_setup.rb [hunk 1]
```diff
@@ -190,4 +190,29 @@
...
+  #           }
+  #         ]
+  #       },
+  #       client: harvest_client
+  #     )
+  #     invoice.save
+  #     invoice
+  #   end
+  # end
+
```
Comment (chimedpalden) [inline spec/harvesting/harvest_data_setup.rb]: @etagwerker  Do dev's have the permission to create new cassette?

File: lib/harvesting/models/invoice.rb [hunk 2]
```diff
@@ -33,9 +33,18 @@ class Invoice < HarvestRecord
                  :created_at,
                  :updated_at
 
+      modeled client: Client,
+              creator: Client
```
Comment (etagwerker) [inline lib/harvesting/models/invoice.rb]: @chimedpalden I'm not familiar with this endpoint. Isn't the `creator` a `User`?

File: spec/harvesting/harvest_data_setup.rb (hunk 1 above)
Comment (etagwerker) [reply spec/harvesting/harvest_data_setup.rb]: @chimedpalden Yes, you will have to use your own Harvest account to create the cassettes. Cassettes are accepted and encouraged for all tests.

File: spec/harvesting/models/invoice_spec.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  # describe '.get' do
+  #   it 'provides direct access to a specific invoice' do
+  #     invoice = Harvesting::Models::Invoice.get(toto_invoice.id)
+  #     expect(invoice.id).to eq(toto_invoice.id)
+  #     expect(invoice.client.id.to_i).to eq(toto_invoice.client.id.to_i)
+  #     expect(invoice.creator.id.to_i).to eq(toto_invoice.creator.id.to_i)
+  #     expect(invoice.line_items.count).to eq(1)
+  #     expect(invoice.line_items.first.project.id.to_i).to eq(toto_invoice.line_items.first.project.id.to_i)
+  #   end
+  # end
```
Comment (etagwerker) [inline spec/harvesting/models/invoice_spec.rb]: @chimedpalden What's the problem with this scenario?

**Comment by chimedpalden** [comment]
@etagwerker this pr is ready for review again.

**Comment by etagwerker** [comment]
@chimedpalden Could you check the failing spec?

**Comment by chimedpalden** [comment]
> @chimedpalden Could you check the failing spec?

sure

**Pull request closed without merging**